src/sql/workbench/services/accountManagement/browser/accountManagementService.ts

```diff
@@ -399,6 +401,27 @@ export class AccountManagementService implements IAccountManagementService {
 		});
 	}
 
+	/**
+	 * Updates the account list for a provider.
+	 * @param provider Provider to update the account list for
+	 */
+	public async updateAccountList(account: azdata.Account): Promise<void> {
+		const sessions = await this._authenticationService.getSessions(account.key.providerId);
```

Comment:
You should make it explicitly clear in the doc and/or name for this function that it only handles auth sessions - not accountManagement sessions.